File: 01_ProgrammingBasics/_04_LoopsPartOne/_07_SumNumbers.java
package _01_ProgrammingBasics._04_LoopsPartOne;

import java.util.Scanner;

public class _07_SumNumbers {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        int sum = 0;

        // Sum all the numbers for the n loop
        for (int i = 1; i <= n; i++) {
            int currentNum = Integer.parseInt(scan.nextLine());
            sum += currentNum;
        }

        // Print the sum of numbers
        System.out.println(sum);

        scan.close();
    }
}

File: 23_C#Basics/_04_LoopsPartOne/_07_SumNumbers/_07_SumNumbers.cs
using System;

public class _07_SumNumbers
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        int sum = 0;

        // Sum all the numbers for the n loop
        for (int i = 1; i <= n; i++)
        {
            int currentNum = int.Parse(Console.ReadLine());
            sum += currentNum;
        }

        // Print the sum of numbers
        Console.WriteLine(sum);
    }
}
